eureka2-write-server/src/main/java/com/netflix/eureka2/server/service/overrides/LoadingOverridesRegistry.java

```diff
@@ -5,60 +5,72 @@
 import com.netflix.eureka2.utils.rx.RxFunctions;
 import rx.Observable;
 import rx.Scheduler;
+import rx.Subscription;
 import rx.functions.Func1;
 import rx.schedulers.Schedulers;
+import rx.subjects.BehaviorSubject;
 
 import javax.annotation.PreDestroy;
+import javax.inject.Inject;
 import javax.inject.Singleton;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicReference;
 
 /**
- * An in memory version of the overrides module primarily useful for testing purposes with the embedded cluster.
+ * A loading overrides registry that stores overrides in an external source, and periodically loads a global
+ * snapshot to a locally backed hash map. Writes to this registry is passed on to the external overrides source,
```

Comment:
Good catch. Should eagerly update the local map here if the underlying put to the external store is successful.